File: modul-1/Soal10/PRAK110-2410817210016-MUHAMMADRIZKYAKBAR.py
import math

a = 5
t = 12
m = math.sqrt ((a*a)+(t*t))

print("Diketahui :")
print(f"Alas = {a}")
print(f"Tinggi = {t}\n")
print("Jawab:")
print(f"Sisi A  = {a} cm")
print(f"Sisi B  = {t} cm")
print(f"Sisi C = {m : .0f} cm")
print(f"Keliling ={ a+t+m : .0f} cm")
print(f"Luas = {a*t/2 : .0f} cm")

File: modul-1/Soal10/PRAK110-2410817210016-MUHAMMADRIZKYAKBAR.c
#include <stdio.h>
#include <math.h>

int main(){
    int a = 5, t = 12, sA, sB, sC;
    sA = t;
    sB = sqrt ((a*a)+(t*t));
    sC = a;

    printf("diketahui: \nAlas = %d cm\n", a);
    printf("Tinggi = %d cm\n\n", t);
    printf("jawab \n");
    printf("Sisi A = %d cm\n", sA);
    printf("Sisi B = %d cm\n", sB);
    printf("Sisi C = %d cm\n", sC);
    printf("Keliling= %d cm\n", sA + sB + sC);
    printf("Luas = %dcm", (a*t)/2);


}